tools/dependency/release_dates.py

```diff
@@ -132,19 +183,27 @@ def verify_and_print_release_dates(repository_locations, github_instance):
 
 
 if __name__ == '__main__':
-    if len(sys.argv) != 2:
+    # github action will have 3 args and CI will have 2 args
+    if len(sys.argv) < 1 or len(sys.argv) > 3:
```

Comment:
if you add `argparse` you should be able to remove this altogether

Reply:
ye, it can be!